xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -123,14 +128,44 @@ bool CVAAPIContext::EnsureContext(CVAAPIContext **ctx, CDecoder *decoder)
   return true;
 }
 
+namespace
+{
+int find_open_render_node_fd()
```

Comment:
I see. As arguments and technical references don't count, I am really wrong in that discussion anyways. Sorry for disturbing you on your "I am right, no one else is. Else no one has a say besides me" - planet.